nosql/nosql-quiz.md

```diff
@@ -138,116 +138,134 @@
  - [ ] MongoDB
  - [ ] Bigtable
 
-#### Q24. You need to implement the simples possible scalable, in-memory cache for your AWS application. Which service do you select?
```

Comment:
sorry but what is the reason of removing this test? I can't find any related duplication FYI

Reply:
ahh, ok I see it now, good finding! FYI there is only different answers :grimacing: